contrib/kafka/cmd/receive_adapter/main.go

```diff
@@ -94,7 +94,12 @@ func main() {
 
 	stopCh := signals.SetupSignalHandler()
 
-	logger.Info("Starting Apache Kafka Receive Adapter...", zap.Reflect("adapter", adapter))
+	logger.Info("Starting Apache Kafka Receive Adapter...",
+		zap.String("Bootstrap Server", string(adapter.BootstrapServers)),
```

Comment:
Remove the space here. That makes the field names easier to use in search queries.

@yanweiguo do you have any guidance on casing for field names in logs? Should this be `BootstrapServer`, `bootstrapServer`, or `bootstrap_server`? I think I prefer `bootstrapServer` or `bootstrap_server`.

Reply:
I'll use your suggested bootstrapServer for now (also keeping relatively inline with the conventions inside the receive_adapter) unless we have a more preferred logging convention specified elsewhere.

Reply:
I'll also remove the casts here as well

Reply:
FWIW, the CCP, over in eventing repo, uses `bootstrap_servers` in the config map for config:

https://github.com/knative/eventing/blob/master/contrib/kafka/config/kafka.yaml#L111

Reply:
[Here](https://github.com/knative/pkg/blob/master/logging/logkey/constants.go) are the log keys we are currently using in logs. I don't think we have a strict naming policy for logs currently. Personally I prefer `bootstrap_server`.

cc @mdemirhan

Reply:
Sure, changed to `bootstrap_server` in e5053ba